README.md

```diff
@@ -66,6 +66,9 @@ property :name, default: 'Paladin'
 - `render_nil` to render or skip nil value (default is false):
 ```ruby
 # will render { name: nil } if name is nil
+```
+- `render_if_key_found` to render or skip if key not found (default is false):
```

Comment:
hmm I think it's in wrong line

Readme before:
<img width="507" alt="image" src="https://github.com/paladinsoftware/simple_representer/assets/719734/5ebc8f3c-730f-41b2-bfa0-d18837b6e64a">

Readme after you changes:
<img width="639" alt="image" src="https://github.com/paladinsoftware/simple_representer/assets/719734/b7803689-822b-4ba4-b8b8-730e67f38ec2">

Soo

```ruby
property :name, render_nil: true
```

this line should be just after 

```ruby
# will render { name: nil } if name is nil
```

Right?

Reply:
oh Thanks!!! @wojtowicz